docs/source/algorithms/RefineSatellitePeaks-v1.rst

```diff
@@ -14,6 +14,15 @@ takes a :ref:`MDWorkspace <MDWorkspace>` of experimental data, a
 integer HKL, and another PeaksWorkspace containing a subset of peaks found at
 fractional HKL. 
 
+.. figure:: /images/RefineSatellitePeaks-satellites.png
```

Comment:
I think your image link is wrong, you need .. at the start, so ../images/RefineSatellitePeaks-satellites.png